packages/Parallel/Application/ParallelFileProcessor.php

```diff
@@ -125,6 +128,18 @@ public function process(
             ++$systemErrorsCount;
             $reachedSystemErrorsCountLimit = true;
             $this->processPool->quitAll();
+
+            // This sleep has to be here, because event though we have called $this->processPool->quitAll(),
+            // it takes some time for the child processes to actually die, and if we would delete the offending cache
+            // files right away, they could still write them "back" before they die
+            sleep(1);
```

Comment:
I am interesting with this `sleep(1);` part, I can found a usecase in the past which CTRL+C sometime still keep running other process in the background, and the file keep changing, especially when running from IDE.

Could you cherry pick this part into separate PR? Thank you.

Reply:
Cherrypicked into https://github.com/rectorphp/rector-src/pull/4280